C116 styling

HadarRose opened the pull request:
- Added CSS file
- removed certain styling components from basicRequests.js

Commit:
Basic requirements met

basic styling required added, rebasing to be more consistent going forward

Commit:
Completed Styling version 1.0

completed styling version corresponding to assignment, needs to be approved.

Commit:
timeline now loads automatically with page

Commit:
Styling modifications v1.1

Included several spacing and sizing alterations, rounded icons, and fixed alignments

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+    font-family: 'Times New Roman', Times, serif;
+}
+
+body {
```
don't use tag names unless you absolutely need to

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+    margin-top: 0px;
+}
+
+button {
```
same as above

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+}
+
+body {
+    margin: 60px;
```
is there a better way to write this and the line below?

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+}
+
+.timeline-container {
+    border-style: solid;
```
single line border

HadarRose replied:
Could you clarify this comment please?

HadarRose replied:
Oh, do you mean writing the shorthand border in one line?

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+}
+
+.timeline-container > div.tweet-block:last-child{
+    border-bottom-style: none;
```
nice, that works

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+}
+
+.tweet-block{
+    border-bottom-style: solid;
```
same as above

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,103 @@
+    padding-bottom: 5px;
+}
+
+.message-div > a{
```
same as above regarding tag names

Commit:
Changed button cursor to pointer

kev-chan commented on src/js/basicRequests.js:
```diff
@@ -26,37 +30,53 @@ function getTimeline() {
     };
     xhttp.onerror = function(){ 
-        document.getElementById("timeline-container").innerHTML = "Something went wrong, please contact systems administrator.";
+        let errorDiv = document.createElement("div");
```
spacing but this could be combined with the above code as well

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,104 @@
+/* TODO delete this probably
```
if this isn't needed, just delete vs leaving TODO

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,104 @@
+    --title-margin: 20;
+}*/
+
+* {
```
this works but better to wrap it in an identifier

HadarRose replied:
Just for clarification, you are talking about the '*' which changes the font-family, and not the parameters in the comments I forgot to delete, correct?

kev-chan replied:
yeah

kev-chan commented on src/css/index.css:
```diff
@@ -0,0 +1,104 @@
+    border-color: #c3c3c4;
+    border-width: 1pt;
+    padding: 20px;
+    padding-bottom: 10px;
```
is there a different way to write this and the line above it?

Commit:
CSS Practices fixes

implemented solution with better CSS practices, as well as a few minor styling changes

Commit:
changed link's styling to use class
